### DISCUSSION TITLE
adding flycast builds to obtainium?

### DISCUSSION THREAD

**Discussion by VorgonBattlecruiser** [ImranR98/Obtainium#1353] [Q&A]
Link: https://flyinghead.github.io/flycast-builds/

im so clueless as to how to add that to obtainium, sorry does anyone know?

**Comment by onnyyonn** [comment]
Can you try adding this source? https://github.com/flyinghead/flycast/releases/

**Comment by DarkCrypt** [comment]
> Link: https://flyinghead.github.io/flycast-builds/

Obtainium can't scrape these links because of JavaScript.
